core/finality-grandpa/src/tests.rs

```diff
@@ -566,6 +573,89 @@ fn finalize_3_voters_no_observers() {
 		"Extra justification for block#1");
 }
 
+
+#[derive(Debug, Default)]
+pub struct TestPool {}
```

Comment:
```suggestion
pub struct TestPool;
```

braced empty structs are only for macros.